src/data/machines.js

```diff
@@ -145,175 +145,176 @@ export const ephemeralExternalIpAddressPrice = { // per hour in dollars for all
 // It parses the API and outputs the response data.
 export const regionToPrices = [
   {
-    name: 'ASIA-EAST1', monthlyStandardDiskPrice: 0.04, monthlySSDDiskPrice: 0.17,
+    name: 'ASIA-EAST1', monthlyStandardDiskPrice: 0.04, monthlySSDDiskPrice: 0.17, balancedDiskPrice: 0.1,
```

Comment:
Can we be consistent with naming the props; e.g. by having them all be `monthly*DiskPrice` or `*DiskPrice`?

Reply:
Good catch! I'll get on that